tests/acceptance/actions-test.js

```diff
@@ -48,6 +54,20 @@ test('allows to chain actions', function(assert) {
   });
 });
 
+test('allows to chain aliased actions', function(assert) {
```

Comment:
While this fixes the problem for actions, I think it breaks the behavior for aliasing regular functions or collections where the return value must remain unchanged.

I would suggest to keep `alias` simple and just return the value of the original function and have an option to change the behavior if desired. So if the user wants to return the alias context, then the user should make it explicit in the alias definition `fooClick: alias('foo.click', { returnAliasContext: true })` (or some other name for the option).

We could go even further and have separated helpers: `alias` and `actionAlias`.